Update description of installing providers separately from core (#24454)
mistercrunch/Airflow: docs/apache-airflow/installation/installing-from-pypi.rst

    CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
    pip install "apache-airflow[postgres,google]==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"

Installing/upgrading/downgrading providers separately from Airflow core
=======================================================================

In order to add new features, implement bug-fixes or simply maintain backwards compatibility, you might need

> to install, upgrade or downgrade any of the providers you need. We release providers independently from the
> core of Airflow, so often new versions of providers are released before Airflow is, also if you do not want
> yet to upgrade Airflow to the latest version, you might want to install newly released providers separately.

Reviewer: ```suggestion
to install, upgrade or downgrade any of the providers you need. We release providers independently from
core Airflow, so often new versions of providers are released before Airflow is. You also may might want to
install newly released providers separately without also upgrading your core Airflow version.
```
